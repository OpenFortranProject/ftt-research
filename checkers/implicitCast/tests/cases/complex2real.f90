program test

  COMPLEX :: from 
  REAL :: to
  
  from = (42.4, 33.33)
  to = from
  
  print *, from
  print *, to 

end
