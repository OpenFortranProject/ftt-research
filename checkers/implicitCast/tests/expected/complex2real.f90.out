Warning: environment variable COMPASS_PARAMETERS is not set, trying to use compass_parameters of the current path
 Running Prerequisite SgProject
ImplicitCast: /home/dagit/research/checkers/implicitCast/tests/cases/complex2real.f90:7.2: Implicit cast in Fortran code found:  expr 'from' from complex_4 to real_4
